prac_05/hex_colours.py

```diff
@@ -0,0 +1,11 @@
+COLOUR_NAMES = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "AliceBlue": "	#f0f8ff",
+                "Alizarin crimson": "#e32636", "Amaranth": "#e52b50", "Amber": "#ffbf00", "Beige": "#f5f5dc",
+                "Champagne": "#f7e7ce", "Charcoal": "#36454f", "Cobalt": "#0047ab", "Coffee": "#6f4e37"}
+valid_colours = {name.lower(): colour.lower() for name, colour in COLOUR_NAMES.items()}
```

Comment:
while converting both the colour names and codes to lowercase, I think its only necessary to convert the names. Colour codes are case sensitive .